src/main/scala/org/virtuslab/internal/load/reader/YamlReader.scala

```diff
@@ -83,9 +85,45 @@ class YamlReader(in: CharSequence) extends Reader {
     skipCharacter() // skip double quote
     val scalar = readScalar()
     List(Scalar(scalar, ScalarStyle.DoubleQuoted))
-  }
 
-  private def fetchFoldedValue(): List[Token] = {
+  private def parseBlockHeader(): Unit =
+    while (peek() == Some(' '))
+      skipCharacter()
+
+    if peek() == Some('\n') then skipCharacter()
```

Comment:
Is block header meaningful name from yaml spec? I am afraid that this name is a little bit enigmatic :/

Reply:
Yes, exactly, this name is from yaml spec - [block-header](https://yaml.org/spec/1.2/spec.html#c-b-block-header(m,t))